horse/users.py

```diff
@@ -26,7 +27,12 @@ class User(Resource):
     def get(self, user_pk):
         user = get_user_by_pk(user_pk)
         return jsonify({
-            'following': [jsonify_user(u) for u in user.get_followed_users()]
+            'following': [
```

Comment:
I'd change it to 'followed_users' instead of 'following', so it matches User model